go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -786,6 +786,10 @@ func TestDoltRevisionDbScripts(t *testing.T) {
 				Query:    "use mydb/" + commithash,
 				Expected: []sql.Row{},
 			},
+			{
+				Query:          "select active_branch();",
+				ExpectedErrStr: "working set not found",
```

Comment:
Not a great error message. This should say "error: no branch selected"

Reply:
Tests for this function after `use mydb/mybranch`?

Reply:
They are here: https://github.com/dolthub/dolt/blob/main/go/libraries/doltcore/sqle/enginetest/dolt_server_test.go#L122